<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Driver Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="createdriverbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>321</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Create new driver</string>
    </property>
    <widget class="QPushButton" name="createdriverbtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create New Driver</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cnamelineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cCniclineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cpasswordlineedit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cphonelineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CNIC:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Area/Station:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cstationcombobox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Account_Status</string>
     </property>
    </widget>
    <widget class="QComboBox" name="caccstatuscombobox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="alldrivergrpbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>781</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>All Drivers</string>
    </property>
    <widget class="QTableWidget" name="driverstable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>711</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deletebtn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>230</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete driver</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="updatedriverbox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Update a driver</string>
    </property>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Station_ID</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ustationidcombobox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="uCniclineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="uphonelineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone number:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="unamelineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CNIC:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Driver_id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="udriveridlineedit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="updatedriverbtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Driver</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Account_Status</string>
     </property>
    </widget>
    <widget class="QComboBox" name="uaccstatuscombobox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="dashboardbtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
